src/static/riot/display_statistics/show_stats.tag

```diff
@@ -1,26 +1,13 @@
 <show-stats>
-    <button id="stats-btn" onclick="{ stats_button_clicked }"
-            class="ui black big launch left attached {fixed: show_stats} button">
-        <i class="icon {minus: !show_stats, 'chart bar': show_stats}"></i>
-        <span class="btn-text">Stats</span>
-    </button>
-    <div id="stat-card" show="{ !show_stats }" class="ui fixed card">
-        <div class="header-bg content">
-            <div class="header">By the numbers...</div>
-        </div>
-        <div class="stat-content content">
-            <h4 class="ui sub blue header">{producer_name} brings together</h4>
-            <div class="ui three column grid">
-                <div class="column" each="{ stat in general_stats }" no-reorder>
-                    <div class="ui seven tiny statistics">
-                        <div class="statistic">
-                            <div class="value">
-                                { stat.count }
-                            </div>
-                            <div class="label">
-                                { stat.label }
-                            </div>
-                        </div>
+    <div class="ui six column grid">
+        <div class="column" each="{ stat in general_stats }" no-reorder>
```

Comment:
thank you!

Reply:
should this be: 
```html
<div class="ui six small statistics">
    <virtual each="{stat in general_stats}">
        <div class="statistic">
            <div class="value">
                { stat.count }
            </div>
            <div class="label">
                { stat.label }
            </div>
        </div>
    </virtual>
</div>
```
instead of putting each stat in its own column, let semantic handle the spacing?

Reply:
I misread this as `{{ stat in general_stats }}`, ughhhh, this should be reading from Django Context Processor not from an API endpoint

Reply:
Semantic is deciding to throw the last stat on a new line if I don't mandate that they're columns. 

This has been moved to home.html and the data is provided by a context processor.